src/web/app/components/question-types/question-statistics/rubric-question-statistics.component.html

```diff
@@ -7,9 +7,11 @@
       <span container="body" [ngbTooltip]="'Format: Response percentage (Response count)' + (hasWeights ? '[Weight of option]' : '') + ', e.g. 50% (2) ' + (hasWeights ? '[1.0]' : '')" class="fa fa-info-circle"></span>
     </div>
     <div class="col-sm-8" style="text-align: right;">
-      <label class="form-check-label">
-        <input id="exclude-self-checkbox" type="checkbox" [(ngModel)]="excludeSelf" (ngModelChange)="getTableData()" class="form-check-input"> <span container="body" class="ngb-tooltip-class" ngbTooltip="Excludes giver's responses to himself/herself from statistics">Exclude self evaluation</span>
-      </label>
+      <div class="form-check">
```

Comment:
I believe the `.form-check` class can be placed directly in the above `div` (i.e. the `div` with class `col-sm-8`). Then we don't need to add in the extra `div`